Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14 AS builder
+WORKDIR /src
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly -v -o operator ./cmd/operator
+
+FROM gcr.io/distroless/static
+WORKDIR /root
```

Comment:
you can just do this `COPY --from=builder /bin/operator /bin/operator`
ENTRYPOINT ["/bin/operator", "-cli"]